src/lib/containers/home.tsx

```diff
@@ -21,6 +21,7 @@ type DataSourceRow = {
 }
 
 interface Props extends ViewProperties, RelayProps {
+  trigger1pxScrollHack?: boolean
```

Comment:
Oh, if this can actually be `undefined` (because you marked it as optional) than I guess comparing just to `false` like we do in `componentDidUpdate` could lead to a false positive. In which case changing it to something like this should work:

```js
(!!previousProps.trigger1pxScrollHack) === false
```